### ISSUE TITLE
Implement cross-driver $element->executeJs()

### ISSUE THREAD

**Issue by Thinkscape** [minkphp/Mink#499]
I've found myself in a corner more than once, trying to use the built in search traversal with JS methods. 

One example of that would be interacting with `<select>` augmenting JS scripts (selectable, select2 etc.). In order to work around issues with those, I need to be able to invoke pieces of JS on some elements.

For example:

``` php
// [...] find the field to change
$field->selectOption($option, false);
$this->getSession()->evaluateScript('$("select").trigger("change");');
```

Currently there's no easy method to get reference to the `NodeElement` node in JS. Internally, Selenium2 driver uses `executeJsOnXpath()` method which takes a script template and "injects" the element we want to target. I'd love to be able to do this:

``` php
// [...] find the field to change
$field->selectOption($option, false);
$field->executeJs('$({{ELEMENT}}).trigger("change");');
```

**Comment by stof** [comment]
@Thinkscape ping

**Comment by stof** [comment]
@Thinkscape any news ?

**Comment by Thinkscape** [comment]
No, sorry. The only thing that comes to my mind is xpath generation (similar to chrome dev tools, when you right click and element and select "copy path").
